.github/dependabot.yml

```diff
@@ -7,11 +7,10 @@ updates:
     labels:
       - "skip issue"
       - "skip news"
-    ignore:
-      - dependency-name: "*"
-        update-types:
-          - "version-update:semver-minor"
-          - "version-update:semver-patch"
```

Comment:
dependabot doesn't just update GHA, this looks like will cause deps in Doc/requirements.txt to start pinging us with every micro-release, no?

Reply:
It's not visible in the PR diff, but the [start of this block in the dependabot.yml file](https://github.com/python/cpython/blob/main/.github/dependabot.yml#L3) declares it's only for GitHub Actions.

There is another block with `- package-ecosystem: pip` which handles version bumps for Python dependencies, but it's actually only looking at the /Tools/ directory. And that block doesn't have an `ignore` field, so CPython is currently receiving minor and patch version bumps for those dependencies ([example](https://github.com/python/cpython/pull/104501)).

However, Doc/requirements.txt is currently ignored by dependabot version bumps. (But it still receives [security updates](https://github.com/python/cpython/pull/105368) whenever a dependency has a vulnerability).